README.md

```diff
@@ -1,56 +1,104 @@
+
+[Changelog]: docs/Changelog.md
+[Schematic]: docs/Schematic.pdf
+[Preview]: docs/Preview.png
+[Vendors]: docs/Vendors.md
+[PCB]: cad
+
+[Green Pro Micro]: http://www.40percent.club/2017/09/green-pro-micro.html
+[License]: https://creativecommons.org/licenses/by-sa/4.0/
+
+[Alvaro]: https://github.com/Ariamelon/Alvaro
+[Comet]: https://github.com/vattern/comet
+[Elite-C]: https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4
+[Goldfish-clone-JLCPCB]: https://github.com/JeremyJStarcher/Goldfish-clone-JLCPCB
+
+<!----------------------------------------------------------------------------->
+
 # Goldfish
-The goldfish is a Pro Micro compatible microcontroller designed for use in mechanical keyboards, although it is perfectly suitable for any other hobby electronics project.
-It is based on the same ATMega32U4 chip as the original Pro Micro and can be programmed using the Arduino IDE, or keyboard specific firmware such as TMK or QMK.
+
+This **Pro Micro** compatible board designed for use in `Mechanical Keyboards`, <br>
+based on the same `ATMega32U4` chip and programmable using the **Arduino IDE** <br>
+or a keyboard specific firmware such as `TMK` or `QMK`.
+
+![Preview]
+
+*Of course it is perfectly usable for any other electronics projects as well.*
+
+---
+
+**⸢ [PCB] ⸥ ⸢ [Schematic] ⸥ ⸢ [Vendors] ⸥ ⸢ [Changelog] ⸥**
```

Comment:
Please keep Vendors and Changelog in the README, not as separate files. The README isn't *that* long and I would prefer these to be visible on the GitHub project page